src/Controller/Api/JobOfferController.php

```diff
@@ -62,4 +66,28 @@ public function display(int $jobOfferId): Response
             AbstractNormalizer::GROUPS => ['detail']
         ]);
     }
+
+    /**
+     * @IsGranted("ROLE_COMPANY")
+     * @Route("/api/job-offer", name="api_job_offer_create", methods={"POST"})
```

Comment:
This route should be named /job-offers/ as a POST method

Reply:
There is a lot of discussion about this one. Here are some texts about it https://www.moesif.com/blog/api-guide/api-design-guidelines/#plural-vs-singular-resource-names. Btw I think we need to create something like a guide to keep us on the same page. Thoughts?

Reply:
Plural is okay to me, actually it was a typo.